File: Low Level Design (LLD)/Problems/04. AirlineManagementSystem/Java/service/NotificationService.java
package service;

import booking.Booking;
import user.Passenger;

public class NotificationService {
    public void sendBookingConfirmation(Passenger passenger, Booking booking) {
        System.out.println("Email sent to " + passenger.getEmail() + ": Booking confirmed for " + booking.getBookingId());
    }
}

File: Low Level Design (LLD)/Problems/04. AirlineManagementSystem/Cpp/service/src/NotificationService.cpp
#include "NotificationService.h"

void NotificationService::sendBookingConfirmation(Passenger* passenger, Booking* booking, Flight* flight, Seat* seat) {
    std::cout << "Email: " << passenger->getEmail() << "\n"
              << "Booking id: " << booking->getBookingId() << "\n"
              << "Flight: " << flight->getFlightNumber() << "\n"
              << "Seat No: " << seat->getSeatNumber() << "\n"
              << "Route: " << flight->getRoute() << "\n"
              << "Schedule: " << flight->getSchedule() << "\n";
}
